src/utils/normalize.ts

```diff
@@ -54,36 +52,14 @@ function parseTypedMessage(data: string) {
 }
 
 /**
- * Normalizes the address to a hexadecimal format
+ * Normalizes the address to standard hexadecimal format
  *
  * @param address - The address to normalize.
  * @returns The normalized address.
  */
-function normalizeContractAddress(address: string): Hex | string {
-  if (isStrictHexString(address) && isValidHexAddress(address)) {
-    return address;
+function normalizeContractAddress(address: Hex): Hex {
+  if (address.startsWith('0X')) {
+    return `0x${address.slice(2)}`;
   }
```

Comment:
Normilizing decimal and octal value is removed now, only normalization now is replacing `0X` prefix with `0x`.